src/style/foundation/themes.less

```
@@ -133,15 +138,20 @@ body.theme-dark {
   // Framework
   // ----------------------------------------------------------------------------
   @app-bg: var(--gray-95);
-  @sidebar-bg: var(--gray-100);
-  @sidebar-border-color: var(--gray-95);
-  @sidebar-folder-selected-bg: var(--gray-95);
   @main-color: var(--white);
   @border-color: var(--gray-90);
   @app-bg-secondary: var(--black);
-  @conversation-list-bg-opacity: fade(#000, 78%);
   @input-bar-bg: var(--gray-100);
 
+  // ----------------------------------------------------------------------------
+  // Coversation Sidebar
```

There is also typing :)

```suggestion
  // Conversation Sidebar
```

---

woah, I straight up looked at this thinking "weird, I didn't make that typo twice"